src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../../components/PeopleFilters/PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import {
+  getFilteredPeople,
+  getPeopleWithParents,
+} from '../../helpers/helpers';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const [searchParams] = useSearchParams();
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
You can replace `searchParams.getAll('centuries') || []` with `searchParams.getAll('centuries')` since `getAll` method returns an array (it can be empty but never `null` or `undefined`).